### DISCUSSION TITLE
ODF plot flipped when compared to OIM (EDAX software)

### DISCUSSION THREAD

**Discussion by trudator** [mtex-toolbox/mtex#1627] [Ask Anything] [answered]
Hi folks, 

I find MTEX super helpful and learning to use it. Kudos to all the creators and contributors that have helped create this.

A question raised when comparing the reconstructed ODF (from EBSD) generated ODF and TSL OIM software (EDAX-based), the ODF plot seems to be flipped horizontally. I tried to change the sample symmetry, and crystal symmetry in MTEX but still doesn't show a matched result. I might assume that OIM software is the standard one and there's something wrong with my MTEX code.

Another question is how to change the scale of phi_1 from 0-90 degree? Thanks.

### MTEX Code:
```
CS = {... 
  'notIndexed',...
  crystalSymmetry('432', [3.6 3.6 3.6], 'mineral', 'Austenite_FCC', 'color', [1 0 0]),...
  crystalSymmetry('432', [2.9 2.9 2.9], 'mineral', 'Martensite_BCC', 'color', [0 1 0])};

% plotting convention
setMTEXpref('xAxisDirection','east');
setMTEXpref('zAxisDirection','intoPlane');

% grains reconstruction
grains = calcGrains(ebsd);
% correct for to small grains
grains = grains(grains.grainSize>5);
% compute optimal halfwidth from the meanorientations of grains
psi = calcKernel(grains('Martensite_BCC').meanOrientation)
 
% Plot ODF 
figure();
ori = ebsd('Martensite_BCC').orientations;
ori.SS = specimenSymmetry('triclinic');
odf = calcDensity(ori,'kernel',psi);

plot(odf,'phi2',[0 45]*degree,'linewidth',1,'colorbar','cs','ss','minmax','contourf',1:2:16,'colorRange',[0,16]); 
mtexColorbar ('location','eastOutSide','title','ODF BCC'); % to move the colour bar to below and include a title
setColorRange('equal');
```

### MTEX ODF:
![image](https://user-images.githubusercontent.com/117386810/199897355-91ce5d85-8ef3-4598-a01b-1edfe14de1ee.png)

### TSL OIM ODF:
![image](https://user-images.githubusercontent.com/117386810/199897572-9fa2f4e9-f743-421a-85b2-35dcd9df95a2.png)

**Comment by Tijmenvermeij** [comment, marked as answer]
Did you use the correct "setting" when importing the data? See here: https://mtex-toolbox.github.io/EBSDReferenceFrame.html
The Edax software uses 'setting 2' normally, but 'setting 1' is used as a default when importing with Mtex...
